MeesGame/Gameplay/GameObjects/Tile.cs

```diff
@@ -71,13 +80,18 @@ public override Vector2 Position
 
         public override void Draw(GameTime gameTime, SpriteBatch spriteBatch)
         {
-            if (!(Parent as TileField).FogOfWar || Revealed)
+            base.Draw(gameTime, spriteBatch);
+
+            if (secondarySprite != null && visible)
             {
-                base.Draw(gameTime, spriteBatch);
-                if (secondarySprite != null && visible)
-                {
-                    secondarySprite.Draw(spriteBatch, this.GlobalPosition, origin, drawColor: secondarySpriteColor);
-                }
+                secondarySprite.Draw(spriteBatch, this.GlobalPosition, origin, drawColor: secondarySpriteColor);
+            }
+
+            if ((parent as TileField).FogOfWar)
+            {
+                //the fog@16 spritesheeet is ordered the same way as the walls spritesheet
```

Comment:
Comment doesn't really help (not at all if you've never seen the walls spritesheet)